charts/cluster-api-cluster-openstack/values.yaml

```diff
@@ -66,6 +66,34 @@ openstack:
 #  certificateSANs:
 #  - foo.acme.com
 
+# Registry mirrors to configure, e.g. for Docker Hub
+# The keys are registry names and the values are mirror configurations
+# registryMirrors:
+#   docker.io:
+#     # The upstream URL
+#     # If not given, "https://<registry>" is used, except for docker.io which is
+#     # automatically converted to https://registry-1.docker.io
+#     upstream: https://registry-1.docker.io
+#     # The list of mirror endpoints for the repository
+#     endpoints:
+#       - # The URL for the endpoint
+#         url: https://harbor.myorg.com/v2/dockerhub-cache
+#         # The capabilities for the endpoint
+#         # If not given, the default is ["pull", "resolve"]
+#         capabilities: ["pull", "resolve", "push"]
+#         # Indicates whether to skip TLS verification for the endpoint
+#         # Defaults to false if not given
+#         skipVerify: true
```

Comment:
Tiniest of tiny nitpicks: on the chance that someone copies this example, uncomments the whole thing and changes one or two fields, it'd be good to give this the safe default `false`.